src/Cassandra.IntegrationTests/Core/EmptyColumnTests.cs

```diff
@@ -0,0 +1,275 @@
+﻿//
+//       Copyright (C) 2019 DataStax Inc.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+
+using System.Collections.Generic;
+using System.Linq;
+
+using Cassandra.Data.Linq;
+using Cassandra.IntegrationTests.TestClusterManagement.Simulacron;
+using Cassandra.Mapping;
+
+using NUnit.Framework;
+
+namespace Cassandra.IntegrationTests.Core
+{
+    [TestFixture]
+    public class EmptyColumnTests
+    {
+        [Test]
+        public void Should_ReturnCorrectValue_When_EmptyColumnName()
+        {
+            using (var simulacronCluster = SimulacronCluster.CreateNew(3))
+            {
+                simulacronCluster.Prime(new
+                {
+                    when = new
+                    {
+                        query = string.Format(
+                            "SELECT * FROM system_schema.tables WHERE table_name='{0}' AND keyspace_name='{1}'",
```

Comment:
Nit: we could use string interpolation that is less error prone.

Reply:
done